partition/roles/dhcp/templates/dhcpd.conf.j2

```diff
@@ -0,0 +1,11 @@
+default-lease-time 600;
+max-lease-time 600;
+
+log-facility local7;
+
+subnet {{ dhcp_server_net }} netmask 255.255.255.0 {
+    range {{ dhcp_range_min }} {{ dhcp_range_max }};
+    # Provide routers to set up default gateway for clients.
+    option routers {{ dhcp_server_ip }};
+    option domain-name-servers {{ groups['mgmt_servers'] | map('extract', hostvars, ['switch_mgmt_ip']) | sort | join(', ') if 'mgmt_servers' in groups else dhcp_public_dns_servers_fallback | join(', ') }};
+}
```

Comment:
```suggestion
subnet {{ dhcp_net }} netmask {{ dhcp_netmask }} {
    range {{ dhcp_range_min }} {{ dhcp_range_max }};
    # Provide routers to set up default gateway for clients.
    option routers {{ dhcp_server_ip }};
    # In case of Vagrant DNS must not be resolved via mgmt servers.
    # This is because the setup to provide e.g. metal images via mgmt servers exceeds resources in Vagrant case.
    option domain-name-servers {{ dhcp_dns_servers | join(', ') }};
}
```